test/SILGen/availability_query.swift

```diff
@@ -57,6 +111,13 @@ func testUnreachableVersionAvailable(condition: Bool) {
     doThing() // FIXME-warning {{will never be executed}}
   }
 
+  if #unavailable(OSX 10.0) {
+    doThing() // FIXME-warning {{will never be executed}}
+  } else {
+    doThing() // no-warning
+    return
+  }
```

Comment:
Not sure why there is a FIXME here... Is it a bug that this warning is reported? (If so, please file a bug report and not the bug report number here. Otherwise, you can remove the FIXME)

Reply:
I think what they tried to say here is that checking for availability for a version equal or less than your minimum target should automatically resolve to a boolean like `*` does (resulting in that warning), but no such check exists today and someone could add it. Below this test there are similar cases but they expect no warnings, so I think it might be better to follow how it works today and expect nothing